test/distance_router_test.cc

```diff
@@ -183,19 +188,43 @@ TEST_F(RoutingInTheSameLaneTest, WithDefaultConstraintsReturnsTheLane) {
 }
 
 // The maximum cost of the phase is smaller than the solution's phase cost, so no routes can be found.
-TEST_F(RoutingInTheSameLaneTest, WithConstrainedPhaseCostReturnsEmpty) {
+TEST_F(RoutingInTheSameSegmentTest, WithConstrainedPhaseCostReturnsEmpty) {
   const std::vector<routing::Route> routes = dut_->ComputeRoutes(start_, end_, kSmallPhaseCostConstraint);
 
   ASSERT_TRUE(routes.empty());
 }
 
 // The maximum cost of the route is smaller than the solution's phase cost, so no routes can be found.
-TEST_F(RoutingInTheSameLaneTest, WithConstrainedRouteCostReturnsEmpty) {
+TEST_F(RoutingInTheSameSegmentTest, WithConstrainedRouteCostReturnsEmpty) {
   const std::vector<routing::Route> routes = dut_->ComputeRoutes(start_, end_, kSmallRouteCostConstraint);
 
   ASSERT_TRUE(routes.empty());
 }
 
+// Because the start and end positions are on the same Lane, no lane switch is required thus there is a solution.
```

Comment:
> required thus

"required and thus"

Reply:
[9a6e1ae](https://github.com/maliput/maliput_integration_tests/pull/84/commits/9a6e1aed7b757f0030f7ce6f8c439182228291cf)